src/state-managers/url-state-manager.js

```diff
@@ -1,24 +1,50 @@
 // @flow
 
+function setSearchParam(key: string, value: string) {
+  const searchParams = new URLSearchParams(
```

Comment:
`new URL(window.location.href).searchParams`  returns a `URLSearchParams` object so no need to wrap it in another one :)

https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams

Reply:
Fixed in b0ed945